client/components/Layout/LayoutSubmissionBannerSkeleton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Eventually, when we figure out what a `LayoutSubmissionBanner` looks like, it will delegate its look 'n' feel to this component. Bit of a dead end on the `onSubmitClicked` handler for now.